styles.css

```diff
@@ -1 +1,87 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/* Removes all default margin and padding for every object on the page */
+* {
+    margin: 0;
+    padding: 0;
+}
+
+/* Define all colors used */
+:root {
+    --light-yellow: #fdefdd;
+    --light-red: #ae4449;
+    --dark-red: #8a363a;
+    --text: #ffffff;
+}
+
+/* Some nicer fonts */
+html, body {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 18px;
+    color: var(--text);
+}
+
+.container {
+    background-color: var(--light-yellow);
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+        "item-a"
+        "item-b"
+        "item-c"
+        "item-d"
+        "item-e"
+        "."
+        "item-f";
+    grid-gap: 16px;
+    padding: 16px;
+}
+
+.container > div {
+    border: 2px solid var(--dark-red);
+    background-color: var(--light-red);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#item-a {
+    grid-area: item-a;
+}
+
+#item-b {
+    grid-area: item-b;
+    /* Some padding added so the paragraph in the div looks nicer */
+    padding: 16px;
+}
+
+#item-c {
+    grid-area: item-c;
+}
+
+#item-d {
+    grid-area: item-d;
+}
+
+#item-e {
+    grid-area: item-e;
+}
+
+#item-f {
+    grid-area: item-f;
+}
+
+/* Media query for screens 576px and bigger */
+@media screen and (min-width:  576px) {
+    .container {
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr;
```

Comment:
repeat (3, 1fr); had ook gekund maar again... 1fr 1fr 1fr is sneller typen

Reply:
dus ik reken het goed ;)

Reply:
Thanks! Sneller typen is wel nice idd